installerscript.iss

```diff
@@ -0,0 +1,48 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "USFM Quick Converter"
+#define MyAppVersion "0.0.1"
+#define MyAppPublisher "Wycliffe Associates"
+#define MyAppURL "https://wycliffeassociates.org"
+#define MyAppExeName "Hackathon_Converter.exe"
```

Comment:
I recommend we rename the executable to USFM_Converter, so it's not confusing to our users.

Reply:
done. that would actually be needed for compatibility as well. Good catch, thanks.

Reply:
@PurpleGuitar Can you resolve this change request so @BryanHo10 can merge these changes and we can get the installer built from the dev branch?

Reply:
Yes, approved!  Sorry for the delay.